### ISSUE TITLE
Add support for AWS Certificate Manager

### ISSUE THREAD

**Issue by aliscott** [infracost/infracost#362]
Resource name: AWS Certificate Manager
Link to pricing information: https://aws.amazon.com/certificate-manager/pricing/
Terraform resources: aws_acm_certificate, aws_acmpca_certificate_authority

**Comment by robh007** [comment]
If nobody minds, I can pick this up. I've got a WIP.

```
  aws_acmpca_certificate_authority.my_private_ca                                                                  
  ├─ Private certificate authority                          1  authority     400.0000       0.5479      400.0000  
  ├─ Certificates (1 - 1K)                              1,000  certificates    0.7500       1.0274      750.0000  
  └─ Certificates (1,0001 - 10K)                        9,000  certificates    0.3500       4.3151    3,150.0000  
  Total                                                                                     5.8904    4,300.0000  

```

**Comment by alikhajeh1** [comment]
@dawidmalina FYI this is now available in v0.7.16
